R/ebird_import.R

```diff
@@ -287,15 +287,46 @@ ebird_import_checks <- function(dt) {
       problems <-
         x[x$flag, ] |>
         dplyr::mutate(
-          desc = glue::glue("{county}, {year}, {route_num}")
+          desc = glue::glue("{route}, {year}")
+        )
+
+      if (any(x$flag)) {
+        logger::log_warn("{problems$desc} had {problems$n} checklists, not 1 or 20 checklists.")
+      }
+    })()
+  
+  # Check for duplicate submissions (> 1 submission in year)
```

Comment:
@IJBG - I think this addresses #124. Please confirm.

Reply:
Yeah, perfect check